client-app/core/composables/useNavigations.ts

```diff
@@ -93,6 +95,27 @@ export function useNavigations() {
     }
   }
 
+  function fetchMobileContactOrganizationsMenu() {
+    const { isMultiOrganization, user } = useUser();
+
+    const organizationsMenuItems = user.value?.contact?.organizations?.items?.map<ExtendedMenuLinkType>((item) => ({
+      id: item.id,
+      title: item.name,
+      isContactOrganizationsItem: true,
+    }));
+
+    if (isMultiOrganization.value) {
+      mobileContactOrganizationsMenu.value = {
+        id: "contact-organizations",
+        title: "common.labels.my_organizations",
+        icon: "/static/images/dashboard/icons/company.svg#main",
+        children: organizationsMenuItems,
+      };
+
+      mobileAccountMenuItem.value?.children?.splice(1, 0, mobileContactOrganizationsMenu.value);
```

Comment:
I'm not sure if it's okay to mutate a computed property. Maybe there is another way to achieve what you want here?

Reply:
why don't we do it something like that to avoid mutation of computed value?:

```js
const mobileContactOrganizationsMenu = ref([])
...
const mobileAccountMenuItem = computed<ExtendedMenuLinkType | null>(() =>
  menuSchema.value 
    ? getTranslatedMenuLink(menuSchema.value.header.mobile.account).splice(1, 0,  ...mobileContactOrganizationsMenu,value)
    : null,
);

function getMobileContactOrganizationsMenu() {
...
mobileAccountMenuItem.value = mobileContactOrganizationsMenu.value;
...
}
```